test/typescript/basic/app.ts

```diff
@@ -1,6 +1,6 @@
 import { FastifyPlugin } from 'fastify'
 import { join } from 'path'
-const fastifyAutoLoad = require('../../../')
+import fastifyAutoLoad from '../../../'
```

Comment:
Full set of import tests would be nice, see https://github.com/fastify/fastify-funky/blob/main/index.test-d.ts for an example.